common/install_cuda.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+set -ex
+
+# install CUDA 9.2 in the same container
+wget -q https://developer.nvidia.com/compute/cuda/9.2/Prod2/local_installers/cuda_9.2.148_396.37_linux -O setup
+chmod +x setup
+./setup --silent --no-opengl-libs --toolkit
+rm -f setup
+
+# patch 1
+wget -q https://developer.nvidia.com/compute/cuda/9.2/Prod2/patches/1/cuda_9.2.148.1_linux -O setup
+chmod +x setup
+./setup -s --accept-eula
+rm -f setup
+
+# install CUDA 9.2 CuDNN
+# cuDNN license: https://developer.nvidia.com/cudnn/license_agreement
+mkdir tmp_cudnn && cd tmp_cudnn
+wget -q http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/libcudnn7-dev_7.6.2.24-1+cuda9.2_amd64.deb -O cudnn-dev.deb
+wget -q http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/libcudnn7_7.6.2.24-1+cuda9.2_amd64.deb -O cudnn.deb
+ar -x cudnn-dev.deb && tar -xvf data.tar.xz
+ar -x cudnn.deb && tar -xvf data.tar.xz
+mkdir -p cuda/include && mkdir -p cuda/lib64
+cp -a usr/include/x86_64-linux-gnu/cudnn_v7.h cuda/include/cudnn.h
+cp -a usr/lib/x86_64-linux-gnu/libcudnn* cuda/lib64
+mv cuda/lib64/libcudnn_static_v7.a cuda/lib64/libcudnn_static.a
+ln -s libcudnn.so.7 cuda/lib64/libcudnn.so
+chmod +x cuda/lib64/*.so
+cp -a cuda/include/* /usr/local/cuda/include/
+cp -a cuda/lib64/* /usr/local/cuda/lib64/
+cd ..
+rm -rf tmp_cudnn
+ldconfig
+
+# install CUDA 10.0 in the same container
+wget -q https://developer.nvidia.com/compute/cuda/10.0/Prod/local_installers/cuda_10.0.130_410.48_linux
+chmod +x cuda_10.0.130_410.48_linux
+./cuda_10.0.130_410.48_linux --silent --no-opengl-libs --toolkit
+rm -f cuda_10.0.130_410.48_linux
+
+# install CUDA 10.0 CuDNN
+# cuDNN license: https://developer.nvidia.com/cudnn/license_agreement
+mkdir tmp_cudnn && cd tmp_cudnn
+wget -q http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/libcudnn7-dev_7.6.2.24-1+cuda10.0_amd64.deb -O cudnn-dev.deb
+wget -q http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/libcudnn7_7.6.2.24-1+cuda10.0_amd64.deb -O cudnn.deb
+ar -x cudnn-dev.deb && tar -xvf data.tar.xz
+ar -x cudnn.deb && tar -xvf data.tar.xz
+mkdir -p cuda/include && mkdir -p cuda/lib64
+cp -a usr/include/x86_64-linux-gnu/cudnn_v7.h cuda/include/cudnn.h
+cp -a usr/lib/x86_64-linux-gnu/libcudnn* cuda/lib64
+mv cuda/lib64/libcudnn_static_v7.a cuda/lib64/libcudnn_static.a
+ln -s libcudnn.so.7 cuda/lib64/libcudnn.so
+chmod +x cuda/lib64/*.so
+cp -a cuda/include/* /usr/local/cuda/include/
+cp -a cuda/lib64/* /usr/local/cuda/lib64/
+cd ..
+rm -rf tmp_cudnn
+ldconfig
+
+# # install CUDA 10.1 in the same container
+wget -q https://developer.nvidia.com/compute/cuda/10.1/Prod/local_installers/cuda_10.1.168_418.67_linux.run
```

Comment:
The original download link `https://developer.nvidia.com/compute/cuda/10.1/Prod/local_installers/cuda_10.1.168_418.67_linux.` doesn't work, I changed it to the current one instead.